script/add-codeql-to-path

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+set -eu
+
+cd "$(dirname "$0")/.."
+
+gh extensions install github/gh-codeql
+gh codeql download
+
+export PATH=$PWD:$PATH
```

Comment:
This part is a bit fiddly. I'm afraid that this won't work because when you run this script from your shell it runs in a subprocess that can't affect the environment variables (including PATH) in its parent process.

We've got two options:
1. Make a `script/test` script that does this and then runs `npm test "$@"` once the PATH is correct. This is nice and simple, but doesn't help if you want `codeql` on the PATH for other reasons.
2. You can make a non-executable bash script that you "source" instead using `source script/add-codeql-to-path` or equivalently `. script/add-code-to-path`, which basically behaves the same as if you typed the commands into your shell.